### PR TITLE
Support multiple topic configs

### PR DESCRIPTION
**Pull request by beets**
- created a top-level config folder for both chart config and topic pages (we should rename the former to place explorer..)
- add support for loading configs as protos, and updating URL paths to /topic/<topic_id>/<place_dcid>
- add placeholder topic landing page
- add placeholder climate config (buggy)

### CODE REVIEW

File: server/config/topic_page/climate/USA.textproto [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
```
Comment (shifucun) [inline server/config/topic_page/climate/USA.textproto:1]: 2022

File: server/config/topic_page/poverty/USA.textproto [hunk 2]
```diff
@@ -12,8 +12,14 @@
...
 # limitations under the License.
 
-## This file specifies the config for topic page
+## This file specifies the topic page config for poverty in USA
 
+metadata {
+  topic_id: 'poverty'
+  topic_name: 'Poverty'
+  place_dcid: 'country/USA'
+  place_dcid: 'geoId/06'
```
Comment (shifucun) [inline server/config/topic_page/poverty/USA.textproto:21]: what is geoId/06 used for here?

**Review by shifucun — APPROVED** [review]

File: server/config/topic_page/climate/USA.textproto [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## This file specifies the topic page config for climate in USA
+
+metadata {
+  topic_id: 'climate'
+  topic_name: 'Climate'
+  place_dcid: 'country/USA'
+  place_dcid: 'geoId/06'
```
Comment (chejennifer) [inline server/config/topic_page/climate/USA.textproto:21]: I noticed in the design doc that there is an additional field in metadata for a map of contained_place_types. Is that something we're still going to include? I think it's something we'll need for a few of the tiles like map and scatter

**Review by chejennifer — APPROVED** [review]
thanks!

File: server/config/topic_page/climate/USA.textproto (hunk 1 above)
Comment (beets) [reply server/config/topic_page/climate/USA.textproto:1]: :) done

File: server/config/topic_page/poverty/USA.textproto (hunk 2 above)
Comment (beets) [reply server/config/topic_page/poverty/USA.textproto:21]: we should specify all pages that the topic page works for. i added CA for testing, and also hoping that the config works for both country and state..

File: server/config/topic_page/climate/USA.textproto (hunk 3 above)
Comment (beets) [reply server/config/topic_page/climate/USA.textproto:21]: yes! that's what it was meant for and we should absolutely include it. there are a few other issues to debug with the configs, and i was hoping to keep that separate from this.

**Comment by beets** [comment]
thanks for the reviews!

**Pull request merged**